pyproject.toml

```diff
@@ -67,6 +67,9 @@ line-length = 120
 exclude = ["nominal/_api/*"]
 include = ["nominal/**/*.py", "tests/**/*.py"]
 
+[tool.ruff.lint.pydocstyle]
+convention = "google"
```

Comment:
google uses `:` with no `-----` underline

Reply:
we probs need a separate PR to add this since many functions don't necessarily abide by it. but not opposed!

Reply:
Ah ok I guess it allows the underline. But IIUC numpy uses the underline, google uses the colon: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
(doesn't really matter for now if it's not complaining, we can be more strict later)